db/migrate/20160924010603_create_users.rb

```diff
@@ -6,6 +6,7 @@ def change
       t.string :last_name
       t.string :email
       t.string :password_digest
+      t.string :status, default: "pending"
```

Comment:
I got it! just wanted to finish up that user settings change first, which I just did :)